<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateRoute</class>
 <widget class="QDialog" name="CreateRoute">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>667</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>200</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create route</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Users/Таня/Pictures/Нова папка/8552951.png</normaloff>../Users/Таня/Pictures/Нова папка/8552951.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(15, 163, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" alignment="Qt::AlignLeft">
    <widget class="QLineEdit" name="LErouteDetails">
     <property name="minimumSize">
      <size>
       <width>204</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 2px solid rgb(0, 0, 0);
}


QLineEdit:hover {
    color: #000000;
    border: 2px solid rgb(0, 0, 0);
}</string>
     </property>
     <property name="placeholderText">
      <string>enter route details...</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="PBback">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 3px solid rgb(0, 0, 0);
}


QPushButton:hover {
    color: #000000;
    background-color: rgb(201, 201, 201); 
    border: 2px solid rgb(0, 0, 0);
}
QPushButton {
	font: 14pt &quot;Gabriola&quot;;
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 3px solid rgb(0, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>back</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTableView" name="TVroute">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignLeft">
    <widget class="QLineEdit" name="LEdistance">
     <property name="minimumSize">
      <size>
       <width>204</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 2px solid rgb(0, 0, 0);
}


QLineEdit:hover {
    color: #000000;
    border: 2px solid rgb(0, 0, 0);
}</string>
     </property>
     <property name="placeholderText">
      <string>enter distance...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="PBnext">
     <property name="minimumSize">
      <size>
       <width>204</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 3px solid rgb(0, 0, 0);
}


QPushButton:hover {
    color: #000000;
    background-color: rgb(201, 201, 201); 
    border: 2px solid rgb(0, 0, 0);
}
QPushButton {
	font: 14pt &quot;Gabriola&quot;;
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 3px solid rgb(0, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>Add route</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" alignment="Qt::AlignLeft">
    <widget class="QLineEdit" name="LErouteName">
     <property name="minimumSize">
      <size>
       <width>204</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 2px solid rgb(0, 0, 0);
}


QLineEdit:hover {
    color: #000000;
    border: 2px solid rgb(0, 0, 0);
}</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>enter name route...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QComboBox" name="CBcomplaxity">
     <property name="minimumSize">
      <size>
       <width>204</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 2px solid rgb(0, 0, 0);
}
QComboBox{
	font: 14pt &quot;Gabriola&quot;;
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 3px solid rgb(0, 0, 0);
}</string>
     </property>
     <item>
      <property name="text">
       <string>The complexity of the route</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>для початківців</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>для бувалих</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>для тих хто хоче екстриму</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>для профі</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" alignment="Qt::AlignLeft">
    <widget class="QLineEdit" name="LEprice">
     <property name="minimumSize">
      <size>
       <width>204</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 2px solid rgb(0, 0, 0);
}


QLineEdit:hover {
    color: #000000;
    border: 2px solid rgb(0, 0, 0);
}</string>
     </property>
     <property name="placeholderText">
      <string>enter price...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="PBdelete">
     <property name="minimumSize">
      <size>
       <width>204</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 3px solid rgb(0, 0, 0);
}


QPushButton:hover {
    color: #000000;
    background-color: rgb(0, 202, 0); 
    border: 2px solid rgb(0, 0, 0);
}
QPushButton {
	font: 14pt &quot;Gabriola&quot;;
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 3px solid rgb(0, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>delete</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="CBrouteType">
     <property name="minimumSize">
      <size>
       <width>204</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 2px solid rgb(0, 0, 0);
}
QComboBox {
	font: 14pt &quot;Gabriola&quot;;
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 3px solid rgb(0, 0, 0);
}</string>
     </property>
     <item>
      <property name="text">
       <string>Route type</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>печера</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>вода</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="LEdelete">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 2px solid rgb(0, 0, 0);
}


QLineEdit:hover {
    color: #000000;
    border: 2px solid rgb(0, 0, 0);
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
